packages/@lwc/shared/src/html-element.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+// @ts-ignore
+const supportsElementInternals = typeof ElementInternals !== 'undefined';
```

Comment:
Moved these to a `@lwc/shared` to keep things consistent between `@lwc/engine-dom` and `@lwc/engine-core`.

Reply:
Moved everything to the [`engine-dom`](https://github.com/salesforce/lwc/pull/3695/commits/4c80dd559e0dc55218c24ea323e6f212945c6668)!